Frist Draft

zh-ouie opened the pull request:
Hello, @dragonyanglong, this PR is for me to learn GitHub and the coding standards. Please review and merge.

dragonyanglong commented on people/zhoujie.py:
```diff
@@ -0,0 +1,6 @@
+import numpy as np 
+ 
+#Converts the list of tuples to ndarray
+list_tuples = [(1,2,3),(4,5)] 
```
please check if this line of code meets the PEP-8 code style, as I mentioned in the tutorial. If not, please modify it by pushing a new commit to the same branch (it will show here automatically on the web); if already meet, please click "resolve".

zh-ouie replied:
Sorry@dragonyanglong, After learning the standard of PEP-8, I changed the style of indentation, and added prefixes to the description when committed. Please review and merge again.

dragonyanglong reviewed (commented):
@zh-ouie good job! I left a quick comment to your code. please work on it accordingly.

dragonyanglong commented:
@zh-ouie, when you resolve the comments I left, please @ me again to review the PR and merge.

BTW, you can try writing the git commit messages using numpy doc style (as I suggested in the tutorial), such as: ENH: first submission to learn github. It uses the prefix three letters to distinguish what your work is.

zh-ouie commented:
Sorry@dragonyanglong  , After learning the standard of PEP-8, I changed the style of indentation, and added prefixes to the description when committed. Please review and merge again.

zh-ouie commented:
Sorry, @dragonyanglong , After learning the standard of PEP-8, I changed the style of indentation, and added prefixes to the description when committed. Please review and merge again.

dragonyanglong commented:
merged. well done.
